packages/@okta/vuepress-theme-prose/components/News.vue

```diff
@@ -20,72 +20,144 @@
+          {{ blog.title }}
         </h3>
         <a
           class="news__article-link dont-break-out"
-          href="https://www.youtube.com/watch?v=PiY5HDp0ABI"
+          :href="blog.link"
         >
-          Watch the video
-        </a>
-      </article>
-      <article class="news__article">
-        <p class="news__text dont-break-out">
-          Blog
-        </p>
-        <h3 class="news__article-title dont-break-out">
-          Step-up Authentication in Modern Applications
-        </h3>
-        <a
-          class="news__article-link dont-break-out"
-          href="https://developer.okta.com/blog/2023/03/08/step-up-auth"
-        >
-          Read the blog post
-        </a>
-      </article>
-      <article class="news__article">
-        <p class="news__text dont-break-out">
-          Blog
-        </p>
-        <h3 class="news__article-title dont-break-out">
-          A Secure and Themed Sign-in Page
-        </h3>
-        <a
-          class="news__article-link dont-break-out"
-          href="https://developer.okta.com/blog/2023/01/12/signin-custom-domain"
-        >
-          Read the blog post
-        </a>
-      </article>
-      <article class="news__article">
-        <p class="news__text dont-break-out">
-          Blog
-        </p>
-        <h3 class="news__article-title dont-break-out">
-          Streamline Your Okta Configuration in Angular Apps
-        </h3>
-        <a
-          class="news__article-link dont-break-out"
-          href="https://developer.okta.com/blog/2023/03/07/angular-forroot"
-        >
-          Read the blog post
+          {{ blog.type === BLOG_TYPES.BLOG ? 'Read the blog post' : 'Watch the video' }}
         </a>
       </article>
     </div>
   </section>
 </template>
+
+<script>
+const axios = require('axios');
+const BLOG_TYPES = {
+  BLOG: 'Blog',
+  VIDEO: 'Video'
+};
+
+const initialBlogPosts = [
+  {
+    type: BLOG_TYPES.BLOG,
+    link: 'https://developer.okta.com/blog/2023/04/04/spa-auth-tokens',
+    title: 'How Authentication and Authorization Work for SPAs'
+  },
+  {
+    type: BLOG_TYPES.VIDEO,
+    link: 'https://www.youtube.com/watch?v=PiY5HDp0ABI',
+    title: 'Podcast: Phishing-Resistant Authenticators with Megha Rastogi'
+  },
+  {
+    type: BLOG_TYPES.BLOG,
+    link: 'https://developer.okta.com/blog/2023/03/08/step-up-auth',
+    title: 'Step-up Authentication in Modern Applications'
+  },
+  {
+    type: BLOG_TYPES.BLOG,
+    link: 'https://developer.okta.com/blog/2023/01/12/signin-custom-domain',
+    title: 'A Secure and Themed Sign-in Page'
+  },
+  {
+    type: BLOG_TYPES.BLOG,
+    link: 'https://developer.okta.com/blog/2023/03/07/angular-forroot',
+    title: 'Streamline Your Okta Configuration in Angular Apps'
+  }
+]
+
+export default {
+  name: 'News',
+  data() {
+    return {
+      blogPosts: initialBlogPosts
+    }
+  },
+  created() {
+    this.BLOG_TYPES = BLOG_TYPES;
+  },
+  mounted() {
+    this.getBlogPosts();
```

Comment:
`getBlogPosts` is async, does it block the `mounted` call?
If we add console log right after HTTP request in `getBlogPosts` and another console log right after `this.getBlogPosts();`
```
async getBlogPosts() {
  await axios.get('https://developer.okta.com/feed.xml'); 
  console.log('recieved response')
}
```
```
mounted() {
    this.getBlogPosts();
    console.log('after getBlogPosts');
}
```

Which message you'll get first? If it is 'after getBlogPosts' then `mounted` function does not wait for `this.getBlogPosts()` and depending on timings we can serve old news or new news for the same user.